go.mod

```diff
@@ -5,14 +5,13 @@ go 1.14
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/alessio/shellescape v1.4.1
-	github.com/evanphx/json-patch/v5 v5.2.0
-	github.com/mattn/go-isatty v0.0.12
-	github.com/pelletier/go-toml v1.8.1
+	github.com/evanphx/json-patch/v5 v5.5.0
+	github.com/mattn/go-isatty v0.0.13
+	github.com/pelletier/go-toml v1.9.3
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.1.1
+	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
-	k8s.io/apimachinery v0.20.2
+	k8s.io/apimachinery v0.21.2
```

Comment:
we should really look at dropping this dependency. staging repos are not fun to have as transitive deps and we're not using much from here.